src/sections/Meshmap/features/index.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
```

Comment:
Please remove these disable comments and fix the eslint issues if any, in the new files added.
You can check using `npm run checklint`